samples/msal-react-samples/b2c-sample/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Msal React B2C Sample</title>
+    <title>React App</title>
```

Comment:
Why was this change made?

Reply:
I've started taking the react-18 sample as template, and then added B2C integration on top of it (later I went back to v 17 to avoid some issues with the ci/cd pipeline but nevermind). This must be left from that. Do you think I should restore the title?